src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier;
+    private final Random random;
+
+    public Lottery(ColorSupplier colorSupplier, Random random) {
+        this.colorSupplier = colorSupplier;
+        this.random = random;
+    }
+
+    public Ball getRandomBall() {
+        return new Ball(getRandomColor(), getRandomNumber());
+    }
+
+    private Color getRandomColor() {
+        return Color.valueOf(colorSupplier.getRandomColor());
+    }
+
+    private int getRandomNumber() {
+        return random.nextInt(100) + 1;
```

Comment:
```suggestion
        return random.nextInt(MAX_BALL_NUMBER) + 1;
```
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/e2dee4c5-24be-4b1a-b4b3-3c92daf36d96)